ReIndex/index.js

```diff
@@ -2,7 +2,20 @@ const azureSearchRequest = require('../lib/AzureSearchRequest');
 
 module.exports = async function reindex(context) {
   const indexName = context.bindings.parameters.indexName;
+  const indexerName = context.bindings.parameters.indexerName;
+  const searchApiVersion = context.bindings.parameters.searchApiVersion;
   const indexDefinition = context.bindings.parameters.indexDefinition;
-  await azureSearchRequest(`indexes/${indexName}`, 'put', JSON.stringify(indexDefinition));
-  await azureSearchRequest(`indexers/${indexName}/run`, 'post');
+  try {
+    await azureSearchRequest(`indexes/${indexName}`, {
+      body: JSON.stringify(indexDefinition),
+      method: 'put',
+      searchApiVersion,
+    });
+    await azureSearchRequest(`indexers/${indexerName}/run`, {
+      method: 'post',
+      searchApiVersion,
+    });
+  } catch (e) {
+    throw Error(`Could not run indexer '${indexerName}' (404 - "Not Found")`);
```

Comment:
Is this error always going to be a 404? Could the error include the message from the exception `e` so it will be more general?